### ISSUE TITLE
Resolve CodeQL Alert 41: Missing variable declaration

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#5706]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/7f0c132c96f71230b8935759e1f8711ccb340c0f/CONTRIBUTING.md).

### Overview
As developers. we need to analyze [CodeQL query alert 41](https://github.com/hackforla/website/security/code-scanning/41) and to either recommend dismissal of the alert or update the code to resolve the alert.    

### Action Items
- [x] DO NOT DISMISS ANY ALERTS.  Dismissal of alerts should be done by dev leads only after review of the recommendation
- [x] Browse to the link in the next Action Item and read the contents.  Click "See More" to view Recommendations, Examples and References.  
- [x] https://github.com/hackforla/website/security/code-scanning/41
- [x] In a comment in this issue, add your analysis and recommendations.  The recommendation can be one of the following: `dismiss as test`, `dismiss as false positive`, `dismiss as won't fix`, or `update code`.  An example of a 'false positive' is a report of a JavaScript syntax error that is caused by markdown or liquid symbols such as `---` or `{%`.  
- [x] Apply the label `ready for dev lead` then move the issue to `Questions/In Review`
- [x] If the recommendation is to update code:
   - [x] create an issue branch and proceed with the code update
   - [x] Using docker, test `/design-system/components/button.html` locally and confirm that the script behaves identically to the existing web page: [Design System Components - Buttons page](https://www.hackforla.org/design-system/components/button.html)
   - [x] proceed with pull request in the usual manner 

### For merge team/dev lead
- [ ] If recommendation to dismiss is approved, dismiss the alert with a comment, then close the issue as completed.
- [ ] When this issue is closed please check off the dependency (under "Analysis Issues") in #5005.  If all analysis issues are closed, close #5005 as completed.  
  
### Resources/Instructions
- [GitHub CodeQL documentation](https://codeql.github.com/docs/codeql-overview/about-codeql/)
- [Design System Components - Buttons page](https://www.hackforla.org/design-system/components/button.html)
- This issue is part of #5005

**Comment by jphamtv** [comment]
Availability for this week: Monday-Friday 12–2 PM, 8–10 PM (PST) 
My estimated ETA for completing this issue: 11/29/23

**Comment by jphamtv** [comment]
The code alert was triggered because the 'i' variable in the for loop is not declared, making it a global variable by default. 

I suggest updating the code to declare the 'i' variable within the loop. This will make it block-scoped and help avoid any future bugs or alerts. 

- I've created an issue branch to update the code. Pull request: https://github.com/hackforla/website/pull/5940
